packages/sit-onyx/src/components/OnyxToastProvider/OnyxToastProvider.stories.ts

```diff
@@ -0,0 +1,86 @@
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryObj } from "@storybook/vue3";
+import OnyxButton from "../OnyxButton/OnyxButton.vue";
+import OnyxToastProvider from "./OnyxToastProvider.vue";
+import { useToast } from "./useToast";
+
+const codeTemplate = `
+  <div>
+    <OnyxButton label="Show toast" @click="showExampleToast" />
+    <OnyxToastProvider />
+  </div>
+`;
+
+/**
+ * Toast provider that is used to display toasts inside the application.
```

Comment:
A thought from when I looked into approving the Toast ticket:


Could you also add such a top description in 
https://storybook.onyx.schwarz/?path=/docs/support-toast--docs
to point out that it is recommended to be used in the ToastProvider?